performance/performance_tuning.rst

```diff
@@ -31,57 +31,59 @@ To begin the tuning process create a Citus cluster and load data in it. From the
 
 Here is an example of explaining the plan for a particular example query.
 
+.. code-block:: postgresql
+
+  EXPLAIN
+   SELECT date_trunc('minute', created_at) AS minute,
```

Comment:
I used a new table because wikipedia_changes is part of the old hash distributed tutorial which no longer exists. This example is from the new real-time analytics tutorial.

The wikipedia example does still exist as one of the typical real-time schemas in "Determing the Data Model" though, so I guess it's not too weird to use it here. Just wanted to use a consistent set of examples throughout the various pages -- happy to use whichever example you prefer here.